modules/data.atmosphere/tests/testthat/test.load.cfmet.R

```diff
@@ -56,17 +57,10 @@ test_that("load.cfmet throws error if start/end date out of range",{
 })
 
 test_that("load.cfmet enforces lat/lon matching",{
-
-  skip("Broken test #1344")
-
-  expect_is(load.cfmet(met.nc = daily.nc, lat = 39, lon = -88,
-                       start.date = "1951-01-01", end.date = "1951-01-07"),
-            "data.frame")
```

Comment:
NB removal of `expect_is` was intentional -- the same call is already checked in earlier tests.